### PR TITLE
Updated yaml version

### PR DESCRIPTION
**Pull request by swathiswaminathan**
Made the below changes in the PR:
1. READme.md: Updated output of the 'gcloud container describe' command
2. manifests/network-policy-namespaced.yaml: updated the apiVersion from core/v1 to v1.
3. Organized all the .sh files into 'scripts' folder. Updated file paths in script files.
4. terraform/bastion.tf: added local-exec provisioners for script files.
5. terraform/main.tf: Updated deprecated terraform code for private clusters to updated version. Replaced private_clusters field to private_cluster_config block.
6. Updated file paths of .sh files in Makefile.

### CODE REVIEW

**Comment by swathiswaminathan** [comment]
test this please

**Comment by chrislovecnm** [comment]
@swathiswaminathan CI is not happy.  Please look at it

File: terraform/bastion.tf [hunk 1]
```diff
@@ -77,6 +77,7 @@ resource "google_compute_instance" "gke-bastion" {
   // Copy the manifests to the bastion
   provisioner "local-exec" {
     interpreter = ["/bin/bash", "-c"]
+
     command = <<EOF
         READY=""
```
Comment (chrislovecnm) [inline terraform/bastion.tf]: We should have the other sets in here. Take a look at the regular bash scripts

**Comment by swathiswaminathan** [comment]
test this please.

**Comment by chrislovecnm** [comment]
@swathiswaminathan why is this failing in CI?  Please take a look at Jenkins.

File: terraform/bastion.tf [hunk 2]
```diff
@@ -97,4 +98,13 @@ resource "google_compute_instance" "gke-bastion" {
         gcloud compute  --project ${var.project} scp --zone ${var.zone} --recurse ../manifests ${var.ssh_user_bastion}@${var.bastion_hostname}:
 EOF
   }
+
+  provisioner "local-exec" {
+    command = "gcloud compute scp --project ${var.project} --zone ${var.zone} --recurse ../manifests ${var.bastion_hostname}:"
```
Comment (chrislovecnm) [inline terraform/bastion.tf]: Why are we adding this?

File: terraform/bastion.tf (hunk 1 above)
Comment (chrislovecnm) [reply terraform/bastion.tf]: Still did not fix this

File: scripts/generate-tfvars.sh [hunk 3]
```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
```
Comment (chrislovecnm) [inline scripts/generate-tfvars.sh]: Why do we need these new files. I do not want to make major changes now. Just fixing bugs is awesome

Comment (chrislovecnm) [reply scripts/generate-tfvars.sh]: @swathiswaminathan what is the purpose of adding this and the other files?  How are they used in this demo?

**Comment by swathiswaminathan** [comment]
Closed and reopened new PR

**Pull request closed without merging**